contracts/acl/ACLSyntaxSugar.sol

```diff
@@ -85,18 +87,34 @@ contract ACLSyntaxSugar {
 }
 
 
-contract ACLHelpers {
-    function decodeParamOp(uint256 _x) internal pure returns (uint8 b) {
+contract ACLHelpers is ACLParams {
+
+
+    function decodeParamOp(uint256 _x) public pure returns (uint8 b) {
```

Comment:
Why are we converting these 3 functions from `internal` to `public`?

Reply:
So it can be used by composition instead of only by inheritance. Do you think it would impact gaz usage and we should keep them `internal`?

Reply:
Most likely wouldn't impact gas by very much (more or less only on deployment), but wondering how useful this would be. Do you think outside contracts might want access to this functionality, which is only accessible on the ACL?

We should also make sure `encodeParams()` and `encodeParam()` are the same visibility as their decode counterparts.

Reply:
Sorry for the delay. hmm.. it’s been so long that I don’t even remember haha! I guess I wanted the `encodeParam` and `encodeParams` functions public, but it’s not supported yet anyway. I converted them back to `internal` and exposed the `encodeOperator` and `encodeIfElse` functions. The two functions are used in the ACL Interpreter tests and I made `encodeOperator` a little bit more explicit.

Reply:
What do you think if I move the `ACLHelpers` contract into its own `ACLHelpers.sol` file? It would be a breaking change however.

Reply:
Let's mark it as something we should do in the future 👍 .